frontend/src/metabase/lib/click-behavior.js

```diff
@@ -42,7 +41,7 @@ export function getClickBehaviorDescription(dashcard) {
 export function hasActionsMenu(dashcard) {
   // This seems to work, but it isn't the right logic.
   // The right thing to do would be to check for any drills. However, we'd need a "clicked" object for that.
-  return !new Question(dashcard.card).isNative();
+  return dashcard.card.dataset_query?.type !== "native";
```

Comment:
`hasActionsMenu` should probably evaluate to `false` when `dataset_query` is `null`.
We can also inverse the check for shorter notation.

```suggestion  
  return dashcard.card.dataset_query?.type === "query";
```

Reply:
Yeah, this makes sense. I don't know what was I thinking. Sorry @uladzimirdev 

No idea why TS was complaining originally.